.vscode/settings.json

```diff
@@ -29,6 +29,11 @@
     "editor.formatOnSave": true,
     "editor.tabSize": 2
   },
+  // Exclude Specific Files from Auto-Imports
```

Comment:
Do you know if this can go somewhere more global so other language servers can pick up on it?

Reply:
its vscode only as far as i know